examples/grpc-client/main.go

```diff
@@ -37,6 +40,34 @@ func run() error {
 
 	client := runnerv1.NewRunnerServiceClient(conn)
 
+	if *createSession {
+		resp, err := client.CreateSession(context.Background(), &runnerv1.CreateSessionRequest{
+			Envs: os.Environ(),
+		})
+		if err != nil || resp.Session == nil {
```

Comment:
The convention is that if `err` is `nil` then `resp.Session` is never `nil` so typically you don't need to check it.